tests/users/test_login_views.py

```diff
@@ -0,0 +1,64 @@
+from django.contrib.auth.models import User
+from django.test import TestCase
+from django.urls import reverse
+
+
+class AccountsTestCase(TestCase):
+    def setUp(self):
+        User.objects.create_user(username='demo_user', password='demo@123')
```

Comment:
self.user = User.objects.create_user(username='demo_user', password='demo@123')
and use self.user to access this user throught the test case change it in both test case files.

Reply:
changes done